targets/targets.json

```diff
@@ -7876,13 +7871,10 @@
     },
     "MAKERDIARY_NRF52840_MDK": {
         "inherits": ["MCU_NRF52840"],
-        "release_versions": ["5"],
-        "device_name": "nRF52840_xxAA",
         "detect_code": ["1026"],
         "device_has_remove": ["ITM"],
         "macros_add": [
-            "CONFIG_GPIO_AS_PINRESET",
-            "NRF52_ERRATA_20"
```

Comment:
I don't see this set in the parent, why are  we removing it ?

Reply:
It's anyway has been erroneously added here. As PR  #11706 tells the define is incorrect and the correct one has been added already with MCU_NRF52840_DK.